lib/human_player.rb

```diff
@@ -0,0 +1,17 @@
+require_relative "player"
+
+class HumanPlayer < Player
+
+  def initialize(position, console)
+    super(position)
+    @console = console
```

Comment:
Why do we need to pass in one instance of console? We can easily initialize the `Console` class where we need it.